index.js

```diff
@@ -1,2 +1,2 @@
 'use strict';
-module.exports = process.defaultApp || /[\\/]electron-prebuilt[\\/]/.test(process.execPath);
+module.exports = process.defaultApp || /[\\/]electron-prebuilt[\\/]/.test(process.execPath) p || /[\\/]electron[\\/]/.test(process.execPath);
```

Comment:
I actually prefer the current as it makes it easier to remove `electron-prebuilt` at some point far in the future.